testutils/lint: TestNightlyLint failed

testutils/lint.TestNightlyLint [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_LintURLsBazel/14291296?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_LintURLsBazel/14291296?buildTab=artifacts#/) on master @ [559994e230ac3776dd1188e66678cce9205ab345](https://github.com/cockroachdb/cockroach/commits/559994e230ac3776dd1188e66678cce9205ab345):


```
             1538:  https://www.cockroachlabs.com/docs/v24.1/alter-function.html
        https://www.cockroachlabs.com/docs/v24.1/show-columns.html : 404 Not Found
             1232:  https://www.cockroachlabs.com/docs/v24.1/show-columns.html
        https://www.cockroachlabs.com/docs/v24.1/drop-database.html : 404 Not Found
             26:  https://www.cockroachlabs.com/docs/v24.1/drop-database.html
        https://www.cockroachlabs.com/docs/v24.1/show-users.html : 404 Not Found
             1565:  https://www.cockroachlabs.com/docs/v24.1/show-users.html
        https://www.cockroachlabs.com/docs/v24.1/show-backup.html : 404 Not Found
             615:  https://www.cockroachlabs.com/docs/v24.1/show-backup.html
        https://www.cockroachlabs.com/docs/v24.1/create-table.html : 404 Not Found
             1786:https://www.cockroachlabs.com/docs/v24.1/create-table.html
        https://www.cockroachlabs.com/docs/v24.1/show-index.html : 404 Not Found
             1112:  https://www.cockroachlabs.com/docs/v24.1/show-index.html
        https://www.cockroachlabs.com/docs/v24.1/rollback-transaction.html : 404 Not Found
             471:  https://www.cockroachlabs.com/docs/v24.1/rollback-transaction.html
        https://www.cockroachlabs.com/docs/v24.1/create-function.html : 404 Not Found
             952:  https://www.cockroachlabs.com/docs/v24.1/create-function.html
        https://www.cockroachlabs.com/docs/v24.1/configure-zone.html : 404 Not Found
             324:  https://www.cockroachlabs.com/docs/v24.1/configure-zone.html
        https://www.cockroachlabs.com/docs/v24.1/show-default-privileges : 404 Not Found
             561:  https://www.cockroachlabs.com/docs/v24.1/show-default-privileges
        https://www.cockroachlabs.com/docs/v24.1/commit-transaction.html : 404 Not Found
             1425:  https://www.cockroachlabs.com/docs/v24.1/commit-transaction.html
        https://www.cockroachlabs.com/docs/v24.1/show-databases.html : 404 Not Found
             552:  https://www.cockroachlabs.com/docs/v24.1/show-databases.html
        https://www.cockroachlabs.com/docs/v24.1/truncate.html : 404 Not Found
             1402:  https://www.cockroachlabs.com/docs/v24.1/truncate.html
        https://www.cockroachlabs.com/docs/v24.1/show-grants.html : 404 Not Found
             573:  https://www.cockroachlabs.com/docs/v24.1/show-grants.html
        https://www.cockroachlabs.com/docs/v24.1/create-procedure.html : 404 Not Found
             1156:  https://www.cockroachlabs.com/docs/v24.1/create-procedure.html
        https://www.cockroachlabs.com/docs/v24.1/show-create.html : 404 Not Found
             13:  https://www.cockroachlabs.com/docs/v24.1/show-create.html
        https://www.cockroachlabs.com/docs/v24.1/savepoint.html : 404 Not Found
             783:  https://www.cockroachlabs.com/docs/v24.1/savepoint.html
             793:  https://www.cockroachlabs.com/docs/v24.1/savepoint.html
        https://www.cockroachlabs.com/docs/v24.1/drop-function.html : 404 Not Found
             1493:  https://www.cockroachlabs.com/docs/v24.1/drop-function.html
        https://www.cockroachlabs.com/docs/v24.1/create-view.html : 404 Not Found
             914:  https://www.cockroachlabs.com/docs/v24.1/create-view.html
        https://www.cockroachlabs.com/docs/v24.1/alter-index.html : 404 Not Found
             1143:  https://www.cockroachlabs.com/docs/v24.1/alter-index.html
        https://www.cockroachlabs.com/docs/v24.1/grant.html : 404 Not Found
             2143:  https://www.cockroachlabs.com/docs/v24.1/grant.html
        https://www.cockroachlabs.com/docs/v24.1/begin-transaction.html : 404 Not Found
             1803:  https://www.cockroachlabs.com/docs/v24.1/begin-transaction.html
        https://www.cockroachlabs.com/docs/v24.1/alter-type.html : 404 Not Found
             494:  https://www.cockroachlabs.com/docs/v24.1/alter-type.html
        54 errors
    --- FAIL: TestNightlyLint/TestHelpURLs (0.82s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
/cc @cockroachdb/test-eng
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestNightlyLint.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>


Jira issue: CRDB-36472

testutils/lint.TestNightlyLint [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_LintURLsBazel/14306128?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_LintURLsBazel/14306128?buildTab=artifacts#/) on master @ [72646a555214c0705781e440b9df585d5eea9511](https://github.com/cockroachdb/cockroach/commits/72646a555214c0705781e440b9df585d5eea9511):


```
             67:  https://www.cockroachlabs.com/docs/v24.1/show-users.html
        https://www.cockroachlabs.com/docs/v24.1/alter-type.html : 404 Not Found
             110:  https://www.cockroachlabs.com/docs/v24.1/alter-type.html
        https://www.cockroachlabs.com/docs/v24.1/alter-procedure.html : 404 Not Found
             687:  https://www.cockroachlabs.com/docs/v24.1/alter-procedure.html
        https://www.cockroachlabs.com/docs/v24.1/grant.html : 404 Not Found
             2139:  https://www.cockroachlabs.com/docs/v24.1/grant.html
        https://www.cockroachlabs.com/docs/v24.1/show-columns.html : 404 Not Found
             904:  https://www.cockroachlabs.com/docs/v24.1/show-columns.html
        https://www.cockroachlabs.com/docs/v24.1/create-view.html : 404 Not Found
             1636:  https://www.cockroachlabs.com/docs/v24.1/create-view.html
        https://www.cockroachlabs.com/docs/v24.1/show-backup.html : 404 Not Found
             1562:  https://www.cockroachlabs.com/docs/v24.1/show-backup.html
        https://www.cockroachlabs.com/docs/v24.1/show-vars.html : 404 Not Found
             736:  https://www.cockroachlabs.com/docs/v24.1/show-vars.html
             1942:  https://www.cockroachlabs.com/docs/v24.1/show-vars.html
        https://www.cockroachlabs.com/docs/v24.1/show-index.html : 404 Not Found
             996:  https://www.cockroachlabs.com/docs/v24.1/show-index.html
        https://www.cockroachlabs.com/docs/v24.1/create-procedure.html : 404 Not Found
             1994:  https://www.cockroachlabs.com/docs/v24.1/create-procedure.html
        https://www.cockroachlabs.com/docs/v24.1/show-partitions.html : 404 Not Found
             327:  https://www.cockroachlabs.com/docs/v24.1/show-partitions.html
        https://www.cockroachlabs.com/docs/v24.1/create-table.html : 404 Not Found
             2225:https://www.cockroachlabs.com/docs/v24.1/create-table.html
        https://www.cockroachlabs.com/docs/v24.1/drop-function.html : 404 Not Found
             2004:  https://www.cockroachlabs.com/docs/v24.1/drop-function.html
        https://www.cockroachlabs.com/docs/v24.1/show-ranges.html : 404 Not Found
             154:  https://www.cockroachlabs.com/docs/v24.1/show-ranges.html
        https://www.cockroachlabs.com/docs/v24.1/configure-zone.html : 404 Not Found
             1244:  https://www.cockroachlabs.com/docs/v24.1/configure-zone.html
        https://www.cockroachlabs.com/docs/v24.1/upsert.html : 404 Not Found
             1319:  https://www.cockroachlabs.com/docs/v24.1/upsert.html
        https://www.cockroachlabs.com/docs/v24.1/select-clause.html : 404 Not Found
             501:  https://www.cockroachlabs.com/docs/v24.1/select-clause.html
        https://www.cockroachlabs.com/docs/v24.1/drop-database.html : 404 Not Found
             318:  https://www.cockroachlabs.com/docs/v24.1/drop-database.html
        https://www.cockroachlabs.com/docs/v24.1/revoke.html : 404 Not Found
             1124:  https://www.cockroachlabs.com/docs/v24.1/revoke.html
        https://www.cockroachlabs.com/docs/v24.1/truncate.html : 404 Not Found
             250:  https://www.cockroachlabs.com/docs/v24.1/truncate.html
        https://www.cockroachlabs.com/docs/v24.1/begin-transaction.html : 404 Not Found
             1436:  https://www.cockroachlabs.com/docs/v24.1/begin-transaction.html
        https://www.cockroachlabs.com/docs/v24.1/create-function.html : 404 Not Found
             1800:  https://www.cockroachlabs.com/docs/v24.1/create-function.html
        https://www.cockroachlabs.com/docs/v24.1/create-database.html : 404 Not Found
             450:  https://www.cockroachlabs.com/docs/v24.1/create-database.html
        https://www.cockroachlabs.com/docs/v24.1/alter-index.html : 404 Not Found
             1850:  https://www.cockroachlabs.com/docs/v24.1/alter-index.html
        54 errors
    --- FAIL: TestNightlyLint/TestHelpURLs (0.86s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestNightlyLint.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

testutils/lint.TestNightlyLint [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_LintURLsBazel/14316625?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_LintURLsBazel/14316625?buildTab=artifacts#/) on master @ [41da9eccc216413c09cc66ad4c6bfa5f92c506e3](https://github.com/cockroachdb/cockroach/commits/41da9eccc216413c09cc66ad4c6bfa5f92c506e3):


```
             689:  https://www.cockroachlabs.com/docs/v24.1/table-expressions.html
             1089:  https://www.cockroachlabs.com/docs/v24.1/table-expressions.html
             1465:  https://www.cockroachlabs.com/docs/v24.1/table-expressions.html
             1648:  https://www.cockroachlabs.com/docs/v24.1/table-expressions.html
        https://www.cockroachlabs.com/docs/v24.1/begin-transaction.html : 404 Not Found
             76:  https://www.cockroachlabs.com/docs/v24.1/begin-transaction.html
        https://www.cockroachlabs.com/docs/v24.1/create-table-as.html : 404 Not Found
             591:https://www.cockroachlabs.com/docs/v24.1/create-table-as.html
        https://www.cockroachlabs.com/docs/v24.1/drop-table.html : 404 Not Found
             849:  https://www.cockroachlabs.com/docs/v24.1/drop-table.html
        https://www.cockroachlabs.com/docs/v24.1/show-partitions.html : 404 Not Found
             358:  https://www.cockroachlabs.com/docs/v24.1/show-partitions.html
        https://www.cockroachlabs.com/docs/v24.1/upsert.html : 404 Not Found
             1007:  https://www.cockroachlabs.com/docs/v24.1/upsert.html
        https://www.cockroachlabs.com/docs/v24.1/alter-procedure.html : 404 Not Found
             283:  https://www.cockroachlabs.com/docs/v24.1/alter-procedure.html
        https://www.cockroachlabs.com/docs/v24.1/commit-transaction.html : 404 Not Found
             2226:  https://www.cockroachlabs.com/docs/v24.1/commit-transaction.html
        https://www.cockroachlabs.com/docs/v24.1/cluster-settings.html : 404 Not Found
             1414:https://www.cockroachlabs.com/docs/v24.1/cluster-settings.html
             1731:  https://www.cockroachlabs.com/docs/v24.1/cluster-settings.html
        https://www.cockroachlabs.com/docs/v24.1/set-transaction.html : 404 Not Found
             1307:https://www.cockroachlabs.com/docs/v24.1/set-transaction.html
        https://www.cockroachlabs.com/docs/v24.1/show-index.html : 404 Not Found
             2214:  https://www.cockroachlabs.com/docs/v24.1/show-index.html
        https://www.cockroachlabs.com/docs/v24.1/grant.html : 404 Not Found
             1756:  https://www.cockroachlabs.com/docs/v24.1/grant.html
        https://www.cockroachlabs.com/docs/v24.1/show-columns.html : 404 Not Found
             195:  https://www.cockroachlabs.com/docs/v24.1/show-columns.html
        https://www.cockroachlabs.com/docs/v24.1/show-zone-configurations.html : 404 Not Found
             876:  https://www.cockroachlabs.com/docs/v24.1/show-zone-configurations.html
        https://www.cockroachlabs.com/docs/v24.1/rollback-transaction.html : 404 Not Found
             814:  https://www.cockroachlabs.com/docs/v24.1/rollback-transaction.html
        https://www.cockroachlabs.com/docs/v24.1/drop-database.html : 404 Not Found
             785:  https://www.cockroachlabs.com/docs/v24.1/drop-database.html
        https://www.cockroachlabs.com/docs/v24.1/set-vars.html : 404 Not Found
             175:  https://www.cockroachlabs.com/docs/v24.1/set-vars.html
             349:https://www.cockroachlabs.com/docs/v24.1/set-vars.html
             1099:  https://www.cockroachlabs.com/docs/v24.1/set-vars.html
             1116:https://www.cockroachlabs.com/docs/v24.1/set-vars.html
        https://www.cockroachlabs.com/docs/v24.1/delete.html : 404 Not Found
             468:  https://www.cockroachlabs.com/docs/v24.1/delete.html
        https://www.cockroachlabs.com/docs/v24.1/alter-view.html : 404 Not Found
             254:  https://www.cockroachlabs.com/docs/v24.1/alter-view.html
        https://www.cockroachlabs.com/docs/v24.1/revoke.html : 404 Not Found
             1873:  https://www.cockroachlabs.com/docs/v24.1/revoke.html
        https://www.cockroachlabs.com/docs/v24.1/insert.html : 404 Not Found
             244:  https://www.cockroachlabs.com/docs/v24.1/insert.html
        54 errors
    --- FAIL: TestNightlyLint/TestHelpURLs (0.72s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestNightlyLint.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

testutils/lint.TestNightlyLint [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_LintURLsBazel/14327145?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_LintURLsBazel/14327145?buildTab=artifacts#/) on master @ [41da9eccc216413c09cc66ad4c6bfa5f92c506e3](https://github.com/cockroachdb/cockroach/commits/41da9eccc216413c09cc66ad4c6bfa5f92c506e3):


```
        https://www.cockroachlabs.com/docs/v24.1/insert.html : 404 Not Found
             1963:  https://www.cockroachlabs.com/docs/v24.1/insert.html
        https://www.cockroachlabs.com/docs/v24.1/show-columns.html : 404 Not Found
             882:  https://www.cockroachlabs.com/docs/v24.1/show-columns.html
        https://www.cockroachlabs.com/docs/v24.1/create-table.html : 404 Not Found
             644:https://www.cockroachlabs.com/docs/v24.1/create-table.html
        https://www.cockroachlabs.com/docs/v24.1/show-partitions.html : 404 Not Found
             2035:  https://www.cockroachlabs.com/docs/v24.1/show-partitions.html
        https://www.cockroachlabs.com/docs/v24.1/show-backup.html : 404 Not Found
             200:  https://www.cockroachlabs.com/docs/v24.1/show-backup.html
        https://www.cockroachlabs.com/docs/v24.1/show-index.html : 404 Not Found
             33:  https://www.cockroachlabs.com/docs/v24.1/show-index.html
        https://www.cockroachlabs.com/docs/v24.1/show-zone-configurations.html : 404 Not Found
             412:  https://www.cockroachlabs.com/docs/v24.1/show-zone-configurations.html
        https://www.cockroachlabs.com/docs/v24.1/alter-view.html : 404 Not Found
             10:  https://www.cockroachlabs.com/docs/v24.1/alter-view.html
        https://www.cockroachlabs.com/docs/v24.1/show-tables.html : 404 Not Found
             1219:  https://www.cockroachlabs.com/docs/v24.1/show-tables.html
        https://www.cockroachlabs.com/docs/v24.1/backup.html : 404 Not Found
             515:  https://www.cockroachlabs.com/docs/v24.1/backup.html
        https://www.cockroachlabs.com/docs/v24.1/show-grants.html : 404 Not Found
             1836:  https://www.cockroachlabs.com/docs/v24.1/show-grants.html
        https://www.cockroachlabs.com/docs/v24.1/show-users.html : 404 Not Found
             381:  https://www.cockroachlabs.com/docs/v24.1/show-users.html
        https://www.cockroachlabs.com/docs/v24.1/table-expressions.html : 404 Not Found
             294:  https://www.cockroachlabs.com/docs/v24.1/table-expressions.html
             360:  https://www.cockroachlabs.com/docs/v24.1/table-expressions.html
             640:  https://www.cockroachlabs.com/docs/v24.1/table-expressions.html
             953:  https://www.cockroachlabs.com/docs/v24.1/table-expressions.html
             1007:  https://www.cockroachlabs.com/docs/v24.1/table-expressions.html
             1191:  https://www.cockroachlabs.com/docs/v24.1/table-expressions.html
             2099:  https://www.cockroachlabs.com/docs/v24.1/table-expressions.html
        https://www.cockroachlabs.com/docs/v24.1/show-create.html : 404 Not Found
             158:  https://www.cockroachlabs.com/docs/v24.1/show-create.html
        https://www.cockroachlabs.com/docs/v24.1/import-into.html : 404 Not Found
             65:  https://www.cockroachlabs.com/docs/v24.1/import-into.html
        https://www.cockroachlabs.com/docs/v24.1/truncate.html : 404 Not Found
             427:  https://www.cockroachlabs.com/docs/v24.1/truncate.html
        https://www.cockroachlabs.com/docs/v24.1/create-database.html : 404 Not Found
             2087:  https://www.cockroachlabs.com/docs/v24.1/create-database.html
        https://www.cockroachlabs.com/docs/v24.1/drop-table.html : 404 Not Found
             2026:  https://www.cockroachlabs.com/docs/v24.1/drop-table.html
        https://www.cockroachlabs.com/docs/v24.1/create-table-as.html : 404 Not Found
             645:https://www.cockroachlabs.com/docs/v24.1/create-table-as.html
        https://www.cockroachlabs.com/docs/v24.1/alter-type.html : 404 Not Found
             2016:  https://www.cockroachlabs.com/docs/v24.1/alter-type.html
        https://www.cockroachlabs.com/docs/v24.1/rollback-transaction.html : 404 Not Found
             1502:  https://www.cockroachlabs.com/docs/v24.1/rollback-transaction.html
        54 errors
    --- FAIL: TestNightlyLint/TestHelpURLs (0.92s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestNightlyLint.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

testutils/lint.TestNightlyLint [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_LintURLsBazel/14347061?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_LintURLsBazel/14347061?buildTab=artifacts#/) on master @ [431d56840cacdc86638ccb48e9f3bdc16897d341](https://github.com/cockroachdb/cockroach/commits/431d56840cacdc86638ccb48e9f3bdc16897d341):


```
             695:  https://www.cockroachlabs.com/docs/v24.1/table-expressions.html
             928:  https://www.cockroachlabs.com/docs/v24.1/table-expressions.html
             940:  https://www.cockroachlabs.com/docs/v24.1/table-expressions.html
             1584:  https://www.cockroachlabs.com/docs/v24.1/table-expressions.html
        https://www.cockroachlabs.com/docs/v24.1/import-into.html : 404 Not Found
             1529:  https://www.cockroachlabs.com/docs/v24.1/import-into.html
        https://www.cockroachlabs.com/docs/v24.1/savepoint.html : 404 Not Found
             100:  https://www.cockroachlabs.com/docs/v24.1/savepoint.html
             296:  https://www.cockroachlabs.com/docs/v24.1/savepoint.html
        https://www.cockroachlabs.com/docs/v24.1/show-tables.html : 404 Not Found
             1620:  https://www.cockroachlabs.com/docs/v24.1/show-tables.html
        https://www.cockroachlabs.com/docs/v24.1/rollback-transaction.html : 404 Not Found
             957:  https://www.cockroachlabs.com/docs/v24.1/rollback-transaction.html
        https://www.cockroachlabs.com/docs/v24.1/truncate.html : 404 Not Found
             1641:  https://www.cockroachlabs.com/docs/v24.1/truncate.html
        https://www.cockroachlabs.com/docs/v24.1/show-backup.html : 404 Not Found
             1849:  https://www.cockroachlabs.com/docs/v24.1/show-backup.html
        https://www.cockroachlabs.com/docs/v24.1/create-table-as.html : 404 Not Found
             700:https://www.cockroachlabs.com/docs/v24.1/create-table-as.html
        https://www.cockroachlabs.com/docs/v24.1/revoke.html : 404 Not Found
             1062:  https://www.cockroachlabs.com/docs/v24.1/revoke.html
        https://www.cockroachlabs.com/docs/v24.1/select-clause.html : 404 Not Found
             929:  https://www.cockroachlabs.com/docs/v24.1/select-clause.html
        https://www.cockroachlabs.com/docs/v24.1/update.html : 404 Not Found
             64:  https://www.cockroachlabs.com/docs/v24.1/update.html
        https://www.cockroachlabs.com/docs/v24.1/create-index.html : 404 Not Found
             2112:https://www.cockroachlabs.com/docs/v24.1/create-index.html
        https://www.cockroachlabs.com/docs/v24.1/alter-index.html : 404 Not Found
             738:  https://www.cockroachlabs.com/docs/v24.1/alter-index.html
        https://www.cockroachlabs.com/docs/v24.1/restore.html : 404 Not Found
             1812:  https://www.cockroachlabs.com/docs/v24.1/restore.html
        https://www.cockroachlabs.com/docs/v24.1/cluster-settings.html : 404 Not Found
             368:  https://www.cockroachlabs.com/docs/v24.1/cluster-settings.html
             970:https://www.cockroachlabs.com/docs/v24.1/cluster-settings.html
        https://www.cockroachlabs.com/docs/v24.1/show-databases.html : 404 Not Found
             109:  https://www.cockroachlabs.com/docs/v24.1/show-databases.html
        https://www.cockroachlabs.com/docs/v24.1/create-function.html : 404 Not Found
             1982:  https://www.cockroachlabs.com/docs/v24.1/create-function.html
        https://www.cockroachlabs.com/docs/v24.1/commit-transaction.html : 404 Not Found
             170:  https://www.cockroachlabs.com/docs/v24.1/commit-transaction.html
        https://www.cockroachlabs.com/docs/v24.1/drop-procedure.html : 404 Not Found
             2025:  https://www.cockroachlabs.com/docs/v24.1/drop-procedure.html
        https://www.cockroachlabs.com/docs/v24.1/drop-table.html : 404 Not Found
             347:  https://www.cockroachlabs.com/docs/v24.1/drop-table.html
        https://www.cockroachlabs.com/docs/v24.1/grant.html : 404 Not Found
             1693:  https://www.cockroachlabs.com/docs/v24.1/grant.html
        https://www.cockroachlabs.com/docs/v24.1/alter-procedure.html : 404 Not Found
             14:  https://www.cockroachlabs.com/docs/v24.1/alter-procedure.html
        54 errors
    --- FAIL: TestNightlyLint/TestHelpURLs (0.95s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestNightlyLint.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

testutils/lint.TestNightlyLint [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_LintURLsBazel/14366918?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_LintURLsBazel/14366918?buildTab=artifacts#/) on master @ [2ac5657aef0fd878973191079b6f3aadbc9c9266](https://github.com/cockroachdb/cockroach/commits/2ac5657aef0fd878973191079b6f3aadbc9c9266):


```
I240313 05:54:00.949566 12009 (gostd) urlcheck.go:235  [-] 10  Checking https://www.cockroachlabs.com/docs/v24.1/insert.html...
I240313 05:54:00.949666 12006 (gostd) urlcheck.go:235  [-] 11  Checking https://www.cockroachlabs.com/docs/v24.1/show-columns.html...
I240313 05:54:00.949719 11983 (gostd) urlcheck.go:235  [-] 12  Checking https://www.cockroachlabs.com/docs/v24.1/show-users.html...
I240313 05:54:00.949768 12010 (gostd) urlcheck.go:235  [-] 13  Checking https://www.cockroachlabs.com/docs/v24.1/show-zone-configurations.html...
I240313 05:54:00.949828 12007 (gostd) urlcheck.go:235  [-] 14  Checking https://www.cockroachlabs.com/docs/v24.1/show-constraints.html...
I240313 05:54:00.949875 12013 (gostd) urlcheck.go:235  [-] 15  Checking https://www.cockroachlabs.com/docs/v24.1/create-function.html...
I240313 05:54:00.949962 12012 (gostd) urlcheck.go:235  [-] 16  Checking https://www.cockroachlabs.com/docs/v24.1/drop-index.html...
I240313 05:54:00.950005 12016 (gostd) urlcheck.go:235  [-] 17  Checking https://www.cockroachlabs.com/docs/v24.1/update.html...
I240313 05:54:00.950046 12017 (gostd) urlcheck.go:235  [-] 18  Checking https://www.cockroachlabs.com/docs/v24.1/drop-database.html...
I240313 05:54:00.950082 12015 (gostd) urlcheck.go:235  [-] 19  Checking https://www.cockroachlabs.com/docs/v24.1/alter-type.html...
I240313 05:54:00.950114 12014 (gostd) urlcheck.go:235  [-] 20  Checking https://www.cockroachlabs.com/docs/v24.1/create-view.html...
I240313 05:54:01.128440 12178 (gostd) urlcheck.go:235  [-] 21  Checking https://www.cockroachlabs.com/docs/v24.1/alter-view.html...
I240313 05:54:01.151464 12179 (gostd) urlcheck.go:235  [-] 22  Checking https://www.cockroachlabs.com/docs/v24.1/drop-function.html...
I240313 05:54:01.171733 12180 (gostd) urlcheck.go:235  [-] 23  Checking https://www.cockroachlabs.com/docs/v24.1/create-table.html...
I240313 05:54:01.196086 12181 (gostd) urlcheck.go:235  [-] 24  Checking https://www.cockroachlabs.com/docs/v24.1/restore.html...
I240313 05:54:01.198039 12147 (gostd) urlcheck.go:235  [-] 25  Checking https://www.cockroachlabs.com/docs/v24.1/backup.html...
I240313 05:54:01.208022 12182 (gostd) urlcheck.go:235  [-] 26  Checking https://www.cockroachlabs.com/docs/v24.1/savepoint.html...
I240313 05:54:01.210131 12148 (gostd) urlcheck.go:235  [-] 27  Checking https://www.cockroachlabs.com/docs/v24.1/show-create.html...
I240313 05:54:01.212989 12183 (gostd) urlcheck.go:235  [-] 28  Checking https://www.cockroachlabs.com/docs/v24.1/grant.html...
I240313 05:54:01.218095 12149 (gostd) urlcheck.go:235  [-] 29  Checking https://www.cockroachlabs.com/docs/v24.1/show-backup.html...
I240313 05:54:01.252639 12184 (gostd) urlcheck.go:235  [-] 30  Checking https://www.cockroachlabs.com/docs/v24.1/alter-function.html...
I240313 05:54:01.256401 12150 (gostd) urlcheck.go:235  [-] 31  Checking https://www.cockroachlabs.com/docs/v24.1/commit-transaction.html...
I240313 05:54:01.259653 12185 (gostd) urlcheck.go:235  [-] 32  Checking https://www.cockroachlabs.com/docs/v24.1/show-grants.html...
I240313 05:54:01.279411 12151 (gostd) urlcheck.go:235  [-] 33  Checking https://www.cockroachlabs.com/docs/v24.1/import-into.html...
I240313 05:54:01.288912 12152 (gostd) urlcheck.go:235  [-] 34  Checking https://www.cockroachlabs.com/docs/v24.1/show-tables.html...
I240313 05:54:01.294179 12186 (gostd) urlcheck.go:235  [-] 35  Checking https://www.cockroachlabs.com/docs/v24.1/select-clause.html...
I240313 05:54:01.320960 12187 (gostd) urlcheck.go:235  [-] 36  Checking https://www.cockroachlabs.com/docs/v24.1/drop-procedure.html...
I240313 05:54:01.337685 12188 (gostd) urlcheck.go:235  [-] 37  Checking https://www.cockroachlabs.com/docs/v24.1/revoke.html...
I240313 05:54:01.348313 12189 (gostd) urlcheck.go:235  [-] 38  Checking https://www.cockroachlabs.com/docs/v24.1/alter-database.html...
I240313 05:54:01.355765 12153 (gostd) urlcheck.go:235  [-] 39  Checking https://www.cockroachlabs.com/docs/v24.1/alter-index.html...
I240313 05:54:01.360983 12190 (gostd) urlcheck.go:235  [-] 40  Checking https://www.cockroachlabs.com/docs/v24.1/show-default-privileges...
I240313 05:54:01.361218 12154 (gostd) urlcheck.go:235  [-] 41  Checking https://www.cockroachlabs.com/docs/v24.1/show-databases.html...
I240313 05:54:01.376878 12155 (gostd) urlcheck.go:235  [-] 42  Checking https://www.cockroachlabs.com/docs/v24.1/table-expressions.html...
I240313 05:54:01.378578 12191 (gostd) urlcheck.go:235  [-] 43  Checking https://www.cockroachlabs.com/docs/v24.1/alter-table.html...
I240313 05:54:01.394072 12156 (gostd) urlcheck.go:235  [-] 44  Checking https://www.cockroachlabs.com/docs/v24.1/delete.html...
I240313 05:54:01.409663 12157 (gostd) urlcheck.go:235  [-] 45  Checking https://www.cockroachlabs.com/docs/v24.1/configure-zone.html...
I240313 05:54:01.414921 12192 (gostd) urlcheck.go:235  [-] 46  Checking https://www.cockroachlabs.com/docs/v24.1/truncate.html...
I240313 05:54:01.416231 12158 (gostd) urlcheck.go:235  [-] 47  Checking https://www.cockroachlabs.com/docs/v24.1/create-procedure.html...
I240313 05:54:01.416859 12159 (gostd) urlcheck.go:235  [-] 48  Checking https://www.cockroachlabs.com/docs/v24.1/cluster-settings.html...
I240313 05:54:01.430387 12160 (gostd) urlcheck.go:235  [-] 49  Checking https://www.cockroachlabs.com/docs/v24.1/upsert.html...
I240313 05:54:01.442172 12161 (gostd) urlcheck.go:235  [-] 50  Checking https://www.cockroachlabs.com/docs/v24.1/set-vars.html...
I240313 05:54:01.449030 12193 (gostd) urlcheck.go:235  [-] 51  Checking https://www.cockroachlabs.com/docs/v24.1/create-table-as.html...
I240313 05:54:01.457591 12194 (gostd) urlcheck.go:235  [-] 52  Checking https://www.cockroachlabs.com/docs/v24.1/drop-table.html...
I240313 05:54:01.465248 12210 (gostd) urlcheck.go:235  [-] 53  Checking https://www.cockroachlabs.com/docs/v24.1/show-partitions.html...
I240313 05:54:01.479651 12195 (gostd) urlcheck.go:235  [-] 54  Checking https://www.cockroachlabs.com/docs/v24.1/rollback-transaction.html...
I240313 05:54:01.483004 12211 (gostd) urlcheck.go:235  [-] 55  Checking https://en.wikipedia.org/wiki/Cron...
    nightly_lint_test.go:46: https://www.cockroachlabs.com/docs/v24.1/configure-zone.html : 404 Not Found
             213:  https://www.cockroachlabs.com/docs/v24.1/configure-zone.html
        1 errors
    --- FAIL: TestNightlyLint/TestHelpURLs (0.87s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestNightlyLint.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

--- a/pkg/sql/parser/sql.y
+++ b/pkg/sql/parser/sql.y
@@ -1928,7 +1928,7 @@ alter_table_stmt:
 //   USING <var> = COPY FROM PARENT [, ...]
 //   { TO | = } <expr>
 //
-// %SeeAlso: WEBDOCS/configure-zone.html
+// %SeeAlso: WEBDOCS/alter-partition.html
 alter_partition_stmt:
   alter_zone_partition_stmt
 | ALTER PARTITION error // SHOW HELP: ALTER PARTITION

